docs/creating_integration_manifest.md

```diff
@@ -46,6 +47,10 @@ The website containing documentation on how to use your integration. If this int
 
 Dependencies are other Home Assistant integrations that you want Home Assistant to set up successfully prior to the integration being loaded. This can be necessary in case you want to offer functionality from that other integration, like using webhooks or an MQTT connection.
 
+## After dependencies
+
+When `after_dependencies` is present, set up of an integration will wait for the other dependency to be set up before being set up.  
```

Comment:
```suggestion
When `after_dependencies` is present, set up of an integration will wait for the other dependency to be set up before being set up. It will also make sure that the requirements of `after_dependencies` are installed so the integration can be safely imported.  
```